### ISSUE TITLE
Add multilanguage support in the keybinding section in settings.

### ISSUE THREAD

**Issue by programnoir** [programnoir/GDTemplate#11]
Currently, the names of Actions and InputEvents cannot be translated, meaning that the keybinding section of the template is exclusively in English. The main idea to fix this would be to provide translations for the unhidden built-in actions as well as the InputEvents that can be assigned to them.

**Comment by programnoir** [comment]
Doesn't seem like Godot offers built-in translations for InputEvents (a [proposal](https://github.com/godotengine/godot-proposals/issues/7334) is gaining popularity, however). Going to assign this with the difficult tag since it'll take a significant amount of work. At the very least, [Godot's editor](https://github.com/godotengine/godot/blob/master/editor/translations/editor/ja.po) has translations that I can use.

**Comment by programnoir** [comment]
Sweet, found a way to do it. Massively inconvenient, but it can be done.
